core-common/src/main/resources/nakadi.access.log.avsc

```diff
@@ -15,6 +15,10 @@
       "name": "query",
       "type": "string"
     },
+    {
```

Comment:
I am not sure that is possible without introduction of versions :(

Reply:
Is it because the field is added in the middle?  Was it not a concern here: https://github.com/zalando/nakadi/pull/1373/files#diff-c9948493d81e5ba0addbf3d4caf4f33320ba07dd503c7a205ff55063a6e44ac3 ?

Reply:
it was, but avro was disabled ;)

Reply:
with avro it is not possible.